DiaryAha/DiaryAha/Components/GetIdeasView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  GetIdeasView.swift
+//  DiaryAha
+//
+//  Created by Keeley Carrigan on 11/24/24.
+//
+
+import SwiftUI
+
+struct GetIdeasView: View {
+    let backgroundColors = [
+        Color.barberry,
+        Color.freshBlue,
+        Color.orangePop
+    ]
+
+    @State private var entryIdeas: [String] = []
+    @State private var idea = ""
+    @State private var backgroundColor: Color = .clear
+
+    func getIdeas() {
+        if (entryIdeas.isEmpty) {
+            guard let url = URL(string: Constants.getTopicsURL) else {
+                return print("Invalid URL")
+            }
+
+            var request = URLRequest(url: url)
+
+            request.httpMethod = "GET"
+
+            let session = URLSession.shared
+            let task = session.dataTask(with: request) { data, response, error in
```

Comment:
Use the Async/Await equivalent.

Reply:
ok. was using the example from the networking tutorial